.travis.yml

```diff
@@ -19,6 +19,11 @@ matrix:
 cache:
   - pip
 
+branches:
+  only:
+    - master
+    - /v?^\d+\.\d+(\.\d+)?(-\S*)?$/
```

Comment:
what kind of branch names are you expecting here?

Reply:
Tag names like `1.3.4` or `v1.3.4`, occasionally `1.5-beta1`, etc.  It's deliberately broader than the range of version tags we typically use to avoid confusion when something a little out of the ordinary fails to trigger a release.  The net effect of this part of the config file is "only run tests on pushes of master or new release tags" (PR tests are handled separately).

Reply:
I see. Thanks